soh/soh/GameMenuBar.cpp

```diff
@@ -873,6 +877,10 @@ namespace GameMenuBar {
                 UIWidgets::Tooltip("Injects Golden Skulltula total count in pickup messages");
                 UIWidgets::PaddedEnhancementCheckbox("Pull grave during the day", "gDayGravePull", true, false);
                 UIWidgets::Tooltip("Allows graves to be pulled when child during the day");
+                UIWidgets::PaddedEnhancementCheckbox("Blue Fire Arrows", "gBlueFireArrows", true, false);
+                UIWidgets::Tooltip("Allows Ice Arrows to melt red ice");
+                UIWidgets::PaddedEnhancementCheckbox("Sunlight Arrows", "gSunlightArrows", true, false);
+                UIWidgets::Tooltip("Allows Light Arrows to activate sun switches");
```

Comment:
It's probably a good idea to include something in the tooltip to clarify a room reload might be needed when the options are toggled mid-gameplay.

Maybe something like:
```C
UIWidgets::Tooltip(
"Allows Ice Arrows to melt red ice.\n"
"\n"
"When toggling this option in-game, a room reload might be required for the arrows to work properly."
);
```
And then the same for Sunlight arrows of course.